src/main.rs

```diff
@@ -82,7 +82,11 @@ fn main() {
     match git2::Repository::open(repository_path) {
         Ok(_) => { },
         Err(e) => {
-            logger::stderr(format!("Could not open the git repository: {:?}", e));
+            if e.code() == git2::ErrorCode::NotFound {
+                logger::stderr(format!("Could not find git repository at {}", repository_path));
+            } else {
+                logger::stderr(format!("Could not open the git repository: {:?}", e));
+            }
```

Comment:
I thought it might be nicer with pattern-machting and guards, but this introduces a `ref`. You decide

```rust
    match git2::Repository::open(repository_path) {
        Ok(_) => { },
        Err(ref e) if e.code() == git2::ErrorCode::NotFound => {
            logger::stderr(format!("Could not find git repository at {}", repository_path));
            process::exit(1);
        },
        Err(e) => {
            logger::stderr(format!("Could not open the git repository: {:?}", e));
            process::exit(1);
        }
    };
```